content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -14,34 +14,29 @@ Role-based access control (RBAC) is a method of regulating access to computer or
 
 {{% capture body %}}
 
-`RBAC` uses the `rbac.authorization.k8s.io` {{< glossary_tooltip text="API group" term_id="api-group" >}}
-to drive authorization decisions, allowing you to dynamically configure policies
-through the Kubernetes API.
+RBAC authorization uses the `rbac.authorization.k8s.io`
+{{< glossary_tooltip text="API group" term_id="api-group" >}} to drive authorization
+decisions, allowing you to dynamically configure policies through the Kubernetes API.
 
 To enable RBAC, start the {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}
 with `--authorization-mode=RBAC` (this is the default).
 
-## API Overview
+## API objects
```

Comment:
can we preserve the previous heading links? there are many deep links to topics on this page